src/components/Numbers/index.jsx

```diff
@@ -0,0 +1,124 @@
+import React, { Component } from 'react';
+import dotsImage from './img/dots.png';
+import circleImage from './img/circle.png';
+import { charityAPI } from '../../clients';
+import './style.css';
+
+const Number = ({ number, title }) => {
+  return (
+    <div className="statistics-content__item justify-end flex flex-col w-1/4">
+      <span className="statistics-content__item__value text-center tracking-wide text-c200 text-xl font-light font-body leading-loose">
+        {number}
+      </span>
+      <h3 className="statistics-content__item__name mt-3 tracking-wide capitalize font-light text-c100 text-md">
+        {title}
+      </h3>
+    </div>
+  );
+};
+
+class Numbers extends Component {
+  state = {
+    data: {
+      backgroundImage:
+        'https://lh3.googleusercontent.com/qWM9ByJ3Fbz2tPnnbF4paUtl3N5gRCG01DFESqYOoM9tJIHx4fdGH4ccHX6mwPSfzPdvEiNZUfs2m1fNt_msn02AjDAROWbMmLkYPgOZOs5IEt09BtiBNBatgMQ9Hg-qjEIDP5Pt6WY4uImsKrRrm724Kqx0M5ka-bn4k0n_vjQ0hVSDoxr5TYBTixHvuJbFEwvMOjWEYzri_xuKGhgAcHRzu1Rl70e2nwwJUy_34tgn1hC64Y0wzXhrw1Vq9CM32e0ilvoCAUCtPzCO_i187FgFX-_TZ7yJmb4Ry16XblLACFyRz7NzA25YGs264i2xTtH6ZlhsreofKoirHmLE2fMZkp7fq5MzfsBnLkgZpEPmBrL7HXf35L3VXbBN4j3XYypI9sJAFZTMJ0Dbv0dpWOMT1FebLEZrgz3zrAblD5kL1yljUJMPvfkrV8AYSyHfyBiHNAs0T6b4M5q406VH_ovmZPS_i1js3ZXyuC3H-nlrfUKJGlLU_9suRefWbaxVHEDRJdDG6aEgBLKakc231ZMO51ixjdoH34w7I0YIY8Y6n5j8yedFIRTjfaBqcXiDoaEF4cWjfORKmTSty8qBOHoC2Bt_mj9COHOXD6gsoTUSb3BfL0m7Vz4NG1sLipok5V2-cOY8vHZAZ6kAKXGfppqOsrn6MNlA0Ls_S7Oz0DHp3hMJSfmaJ6nkS4AHOhNe9YdWbMpddht_7IYewcUfjI1YWRIAjwn-nyfHp0Nakvbb7CNpLGGL=w1354-h903-no'
+    },
+    loading: true,
+    error: false,
+    errorMessage: ''
+  };
+
+  componentDidMount() {
+    this._getData();
+  }
+
+  _getData = () => {
```

Comment:
😄  Great Work Hassan ( _you prefer to escape from bind bug_) 🤔 
 You prefered to use an arrow function so that `this` inside `_getData` bind to The class and in this case no need to use the constructor method. 

I found this is [Article](https://frontarm.com/james-k-nelson/when-to-use-arrow-functions/) as It shows why we use `arrow function` in react

Reply:
Thank you for sharing resources.
I highly encourage you to take a look at this wonderful article about the `this ` keyword in React
https://www.freecodecamp.org/news/this-is-why-we-need-to-bind-event-handlers-in-class-components-in-react-f7ea1a6f93eb/